inception-recommendation/pom.xml

```diff
@@ -1,253 +1,244 @@
-<!--
-  Copyright 2017
-  Ubiquitous Knowledge Processing (UKP) Lab
-  Technische Universität Darmstadt
+<!-- Copyright 2017 Ubiquitous Knowledge Processing (UKP) Lab Technische 
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that XML files are indented using spaces only and using 2 spaces for an indentation.